modules/catalog/src/main/java/org/apache/ignite/internal/catalog/events/AddColumnEventParameters.java

```diff
@@ -24,25 +24,27 @@
  * Add column event parameters contains descriptors of added columns.
  */
 public class AddColumnEventParameters extends CatalogEventParameters {
-
     private final int tableId;
-    private final List<CatalogTableColumnDescriptor> columnDescriptors;
+    private final List<CatalogTableColumnDescriptor> descriptors;
```

Comment:
Why did you rename it? Old name was fine

Reply:
But not for me.

Reply:
what is wrong with original name?